docs/internet-identity-spec.adoc

```diff
@@ -716,14 +698,39 @@ dfx canister --network mainnet --no-wallet info internet_identity
 
 Once the deployment went through, tag the commit with `mainnet-<date-from-proposal-url>`.
 
+=== Deploying on testnets
 
-==== On testnets
+==== Building the canister for the identity testnet
 
-Once you've built the binary, you can then run (for `messaging`):
+Because we need to fetch the root key for all networks that are not mainnet, we can not use the regular Docker build.
+Instead we build ourselves with the `II_ENV` environment variable set to `development`.
 
 [source,bash]
 ----
-ic-admin --nns-url http://dcs-messaging-13.dfinity.systems:8080/ propose-to-change-nns-canister --test-neuron-proposer --canister-id rdmx6-jaaaa-aaaaa-aaadq-cai --mode upgrade --wasm-module-path internet_identity.wasm
+didc encode '(null)' | xxd -r -p > arg.in
+npm ci
+II_ENV=development dfx build --network identity
+cp target/wasm32-unknown-unknown/release/internet_identity.wasm .
+----
+
+This will create the Wasm file you want to use for the following deployment steps at `./internet_identity.wasm`.
+
+==== Installing on testnets
+
+Submit the proposal to install the canister on our `identity` testnet (replace for other testnets as appropriate):
+[source,bash]
+----
+ic-admin --nns-url "http://[2a00:fb01:400:42:5000:60ff:fed5:8464]:8080/" propose-to-change-nns-canister --test-neuron-proposer --canister-id rdmx6-jaaaa-aaaaa-aaadq-cai --mode reinstall --wasm-module-path internet_identity.wasm --arg arg.in
```

Comment:
I compared to the command I used in our session. I used `--mode install` here. Does that matter or is `reinstall` fine too?

Reply:
> I guess reinstall is just more general in that it reinstalls in case one is installed already?

Yes. It'll wipe all state in the process (compared to doing an upgrade). I think it's preferable to document this command, because it works in more situations, and on the testnet we're not too worried about wiping existing state.